src/main/java/gameboy/gamepack/racinggame/view/InputView.java

```diff
@@ -22,4 +36,20 @@ private static int userIntInput() {
         Scanner scanner = new Scanner(System.in);
         return scanner.nextInt();
     }
+
+    private static String userStringInput() {
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextLine();
+    }
+
+    private static boolean validationRacerName(String name) {
+        if (name.isBlank()) {
+            throw new InvalidRacerNameException("자동차 이름은 공백이 될 수 없습니다.");
+        }
+        if (name.length() > MAX_NAME_SIZE) {
+            throw new InvalidRacerNameException("자동차 이름이 " + MAX_NAME_SIZE + "글자를 초과할 수 없습니다.");
+        }
+        return true;
+    }
```

Comment:
```suggestion
    private static void validateRacerName(String name) {
        if (name.isBlank()) {
            throw new InvalidRacerNameException("자동차 이름은 공백이 될 수 없습니다.");
        }
        if (name.length() > MAX_NAME_SIZE) {
            throw new InvalidRacerNameException("자동차 이름이 " + MAX_NAME_SIZE + "글자를 초과할 수 없습니다.");
        }
    }
```
이럴거면 그냥 void 타입으로 선언하면 될 것 같습니다.
true 값을 리턴하지 않아도 될 것 같아요